__tests__/app.test.js

```diff
@@ -32,3 +33,61 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("200: responds with an object describing all the available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
+        expect(body.endpoints).toEqual(expectedEndpoints);
+        //
```

Comment:
We could simply leave the test like this at this point, rather than doing all the logic underneath. We're showing that the response equals the file contents, which is all we really require 🙂